Popup/Popup.module.less

```diff
@@ -40,6 +48,19 @@
 		border-width: @agate-popup-border-width;
 		border-color: @agate-popup-border-color;
 		border-radius: @agate-popup-border-radius;
+		opacity: @agate-popup-opacity;
+
+		&.top {
+			border-radius: @agate-popup-top-border-radius;
+			margin: @agate-popup-top-margin;
+			max-width: @agate-popup-top-max-width;
+			width: @agate-popup-top-width;

```

Comment:
Rather than setting these sorts of sizing things in variables, let's make the `.top` class a pre-defined shape. The top and left rules aren't variable, so this would only be able to stick to the top left corner and be various widths from there. That seems less useful. However, if we instead specify `right` up above, on line 25, we don't need any of the width props here, and the content can size itself.

If you wanted to be _really fancy_ (and who wouldn't‽), you could set it to anchor in the top center of the screen, and allow the content to dictate the width, up to 100% screen width, so a skin could choose its width and it always stay centered. This is more of a bonus, and not mandatory.